pytorch_pfn_extras/training/manager.py

```diff
@@ -349,20 +337,26 @@ def extend(
 
         if trigger is None:
             trigger = ext.trigger
-        trigger = trigger_module.get_trigger(trigger)
 
         if priority is None:
             priority = ext.priority
 
+        self._extend(
```

Comment:
The normalizations above (line 331 to 343) can be done in ExtensionEntry.

Reply:
Normalization operation requires `self._extensions`.